### DISCUSSION TITLE
Generate types to a different file

### DISCUSSION THREAD

**Discussion by finom** [prisma/prisma#16971] [Q&A]
Hello! We have two different repos for back-end (NestJS) and front-end. `prisma generate` works great bur I don't have access to the types from the front-end repository. Is there a way to export types to a different file?

**Comment by nurul3101** [comment]
Hello @finom 👋 
If I understand correctly, do you want to export the types that are generated by Prisma and use them in your frontend repository?
You can change the location of generated PrismaClient using the Output parameter - [Docs](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#the-location-of-prisma-client).

**Reply by finom** [reply]
Thank you for your response @nurul3101 but I meant some flag that allows to set default output without modifying the prisma file. Also prisma generates too many files and I'd like to generate only types. I know it's not possible right now but maybe an idea for future versions.

**Reply by pantharshit00** [reply]
Hello @finom!

I don't think we cover that use case right now. Can you please open a new [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) about this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
